mosdef_cassandra/core/moves.py

```diff
@@ -676,5 +772,58 @@ def print(self):
             contents += "Box {box}: {max_vol}\n".format(
                 box=box + 1, max_vol=max_vol
             )
+        
+        if self.restricted_type != None:
+            contents += "\nRestricted Insertions (Ang):\n"
+            for box in range(self._n_boxes):
+                for species, (typ, value) in enumerate(zip(self.restricted_type[box], self.restricted_value[box])):
+                    if typ == 'sphere':
+                        contents += "Box {box}, Species {species}: sphere, R = {r_value}\n".format(
+                            box=box + 1, species=species + 1, r_value=value)
+                    elif typ == 'cylinder':
+                        contents += "Box {box}, Species {species}: cylinder, R = {r_value}\n".format(
+                            box=box + 1, species=species + 1, r_value=value)
+                    elif typ == 'slitpore':
+                        contents += "Box {box}, Species {species}: slitpore, z_max = {z_max}\n".format(
+                            box=box + 1, species=species + 1, z_max=value)
+                    elif typ == 'interface':
+                        contents += "Box {box}, Species {species}: interface, z_min = {z_min}, z_max = {z_max}\n".format(
+                            box=box + 1, species=species + 1, z_min=value[0],
+                            z_max=value[1])
+                    else:
```

Comment:
What if a non-supported insertion type gets added (or more likely, someone further down the line updates the other parts of the code to support a new type but doesn't update the print). Can we check that it is indeed `None` and raise a `ValueError` if not?

Reply:
I believe this should be fixed now, since `add_restricted_insertion` checks that the insertion type is valid.  All of the restricted insertion types are currently supported, but I can add a `ValueError` in the print statement if you'd like as well.